development/mimir-microservices-mode/config/mimir.yaml

```diff
@@ -183,6 +185,11 @@ limits:
   ha_cluster_label: ha_cluster
   ha_replica_label: ha_replica
   ha_max_clusters: 10
+  
+  cost_attribution_labels: "container"
```

Comment:
This should be an array, right? Not a single string.

Reply:
it would be string seperated by comma, would update the config to make it more clear.

Reply:
updated and tested in fbb2fac8a1b1f52af2fabf1ccdda683592ed22f7

Reply:
This is YAML, why are we using a comma separated string here? I also see it being defined as a slice (comma separated string slice) type in go. Are you sure this is working correctly? I don't really see any tests in that commit that would suggest that this is not parsed as a single `container,instance` label.

Reply:
i ran the service config in order to tested it, this test should covered it https://github.com/grafana/mimir/pull/10269/files#diff-8f7093a5156899c67bddcb606deac6ee65300ffa0e52497045243995b1a18740R1081, i am checking whether I can add more tests

Reply:
I think i would just remove it, i use this to test in local, it is not part of the commit.

Reply:
<img width="1070" alt="image" src="https://github.com/user-attachments/assets/bdc72680-525f-49d9-a634-7aec9a1e84f9" />
the parse works on this config